pygmt/tests/test_coast.py

```diff
@@ -14,60 +13,37 @@ def test_coast():
     """
     fig = Figure()
     fig.coast(
-        R="-30/30/-40/40",
-        J="m0.1i",
-        B=5,
-        I="1/1p,blue",
-        N="1/0.25p,-",
-        W="0.25p,white",
-        G="green",
-        S="blue",
-        D="c",
-        A=10000,
+        region="-30/30/-40/40",
+        projection="m0.1i",
+        frame=5,
+        rivers="1/1p,blue",
+        borders="1/0.25p,-",
+        shorelines="0.25p,white",
+        land="green",
+        water="blue",
+        resolution="c",
+        area_thresh=10000,
     )
     return fig
 
 
-@check_figures_equal()
+@pytest.mark.mpl_image_compare
 def test_coast_iceland():
     """
-    Test passing in R as a list.
+    Test passing in region as a list.
     """
-    fig_ref, fig_test = Figure(), Figure()
-    # Use single-character arguments for the reference image
-    fig_ref.coast(R="-30/-10/60/65", J="m1c", B="", G="p28+r100")
-    fig_test.coast(
-        region=[-30, -10, 60, 65], projection="m1c", frame=True, land="p28+r100"
-    )
-    return fig_ref, fig_test
+    fig = Figure()
+    fig.coast(region=[-30, -10, 60, 65], projection="m1c", frame=True, land="p28+r100")
+    return fig
 
 
-@check_figures_equal()
+@pytest.mark.mpl_image_compare
 def test_coast_aliases():
```

Comment:
This is one of the top 3 slowest tests as reported in #840, can you try and speed it up? E.g. using resolution="c" for crude? Maybe also plot a smaller geographical region.

Reply:
Done 2f7ba86 and 0fcbdfc